.github/workflows/tests.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     - name: Lint with pycodestyle
       run: pycodestyle --show-source capa/ scripts/ tests/
     - name: Check types with mypy
-      run: mypy --config-file .github/mypy/mypy.ini capa/ scripts/ tests/
+      run: mypy --config-file .github/mypy/mypy.ini --check-untyped-defs capa/ scripts/ tests/
```

Comment:
this flag enables mypy checking on functions that would have been otherwise skipped (they don't have type signatures so mypy saves some effort by default). this results in more complete checks and a bunch of fixes we need here.